Gemfile.lock

```diff
@@ -33,11 +33,11 @@ GEM
       debug_inspector (>= 0.0.1)
     builder (3.2.3)
     byebug (11.0.1)
-    chef (15.0.300)
+    chef (15.1.36)
```

Comment:
This PR is going out in a minor version bump of Chef Workstation, correct? Make sure we're not going to have multiple `chef` gem versions bundled

Reply:
This matches the version that we currently pull into ChefDK, which is where WS sources it from.